src/js/components/ColumnChooserMenu.js

```diff
@@ -12,13 +11,66 @@ import CloseButton from "./CloseButton"
 import Columns from "../state/Columns"
 import TableColumns from "../models/TableColumns"
 import dispatchToProps from "../lib/dispatchToProps"
+import Viewer from "../state/Viewer"
+import styled from "styled-components"
+import Checkbox from "./common/Checkbox"
+import usePopupMenu from "./hooks/usePopupMenu"
+import DropdownArrow from "../icons/DropdownArrow"
+import type {ColumnHeadersViewState} from "../state/Viewer/types"
+
+const ControlListItem = styled.li`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 4px 36px;
+  cursor: default;
+
+  p {
+    text-decoration: underline;
+    color: ${(props) => props.theme.colors.havelock};
+  }
+`
+
+const ColumnListItem = styled.li`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 4px 36px;
+  cursor: default;
+  ${(props) => props.theme.typography.labelNormal}
+
+  &:hover {
+    background-color: ${(props) => props.theme.colors.ivory};
+  }
```

Comment:
Now that is a checkbox, I think the hover state is a bit distracting. I'd be fine removing it.